test/fixtures/packages-no-comments.dhall

```diff
@@ -0,0 +1,8 @@
+let upstream =
+      https://github.com/purescript/package-sets/releases/download/psc-0.13.3-20190920/packages.dhall sha256:53873cf2fc4a343a41f335ee47c1706ecf755ac7c5a336e8eb03ad23165dfd28
```

Comment:
Having this fixture will add maintenance burden when bumping package set versions. Suggestions how to avoid it are welcome :-)

Reply:
A simple alternative that could be used instead of having this fixture could be something like this code directly in the test:
```haskell
config <- readTextFile "spago.dhall"
config `shouldNotSatisfy` (Text.isInfixOf "{-") -- comments are not present
config `shouldSatisfy` (Text.isInfixOf "dependencies =") -- but some dhall stuff is present
```

Reply:
Yep, this last idea sounds great!

Reply:
Ok, implemented.